tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/settings_form.py

```diff
@@ -134,6 +137,11 @@ def tap_back_from_import_contacts(self):
         header.tap(25, 25)
         Wait(self.marionette).until(expected.element_not_displayed(header))
 
+    def tap_set_ice_contact(self):
+        self.marionette.find_element(*self._set_ice_contact_button_locator).tap()
+        GaiaApps(self.marionette).switch_to_displayed_app()
```

Comment:
This is not necessary if this logic is there in the `SetIceContacts` class as I described earlier.